app/src/main/java/com/jxc/app/models/User.java

```diff
@@ -2,9 +2,7 @@
 
 import java.util.List;
 
-import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.NoArgsConstructor;
+import lombok.*;
```

Comment:
On va essayer d'éviter les import * . Automatiquement, IntelliJ te converti 5 imports ou plus d'une même librairie en import * , à changer dans les options d'IntelliJ :
Settings > Editor > Code Style > Java > Imports > Class count to use import with ''*".